docs/user_guide/assessment.rst

```diff
@@ -123,7 +123,9 @@ metrics simultaneously:
 .. doctest:: assessment_metrics
     :options:  +NORMALIZE_WHITESPACE
 
-    >>> multi_metric = MetricFrame({'precision':skm.precision_score, 'recall':skm.recall_score},
+    >>> multi_metric = MetricFrame({'precision':skm.precision_score,
+                                    'recall':skm.recall_score,
+                                    'count': count},
```

Comment:
Don't you need to import this first?

Reply:
I think it's imported from `fairlearn.metrics` [here] (https://github.com/fairlearn/fairlearn/blame/8a76289ed54353872b590c790df8de7bd03ba7e3/docs/user_guide/assessment.rst#L7). Is there somewhere else that it needs to be imported?

I built the docs locally, and it seemed to work.

Reply:
That's not an import AFAIK. At least I haven't seen it work that way, but then this wouldn't be the first time I learn something new 😆 Regardless, it would be nice to have it explicitly mentioned where `count` comes from.